### PR TITLE
Added functionality for displaying and updating when tasks and projects are last changed

### PR DESCRIPTION
**Pull request by yvindo**
- Added parameters in types and mock data
- Added fields for displaying last change date
- Added automatic update of last change values in store on update

### CODE REVIEW

File: packages/adminpanel-v2/src/lib/mock/customers.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const Projects: TProject[] = [
 		projectNumber: 1,
 		startDate: new Date('December 17, 1995 03:24:00'),
 		customer: 1,
+		changedDate: new Date('December 17, 1995 03:24:00').toLocaleString('en-GB', { timeZone: 'UTC' }).split(',')[0],
```
Comment (Fordiboii) [inline packages/adminpanel-v2/src/lib/mock/customers.ts:33]: Put string formatting in a function

**Comment by Fordiboii** [comment]
La igjen en liten kommentar, ser bra ut ellers!

File: packages/adminpanel-v2/src/lib/mock/customers.ts (hunk 1 above)
Comment (yvindo) [reply packages/adminpanel-v2/src/lib/mock/customers.ts:33]: Added a formatting function in the store. Didn't change anything in the mock data, as this is just temporary test data.

File: packages/adminpanel-v2/src/stores/CustomerStore.ts [hunk 2]
```diff
@@ -150,10 +150,12 @@ function createCustomers() {
 		setCustomers(customers);
 	};
 
+	const getNewDateString = () => {return new Date().toLocaleString('en-GB', { timeZone: 'UTC' }).split(',')[0]}
```
Comment (Fordiboii) [inline packages/adminpanel-v2/src/stores/CustomerStore.ts:153]: Nice! Here you can even remove the curly brackets if you want to be super smooth and do `getNewDateString = () => (...)`

**Review by Fordiboii — APPROVED** [review]

**Pull request merged**